velox/docs/functions/presto/array.rst

```diff
@@ -2,6 +2,14 @@
 Array Functions
 =============================
 
+.. function:: all_match(array(T), function(T, boolean)) → boolean
```

Comment:
Functions in the documentation should appear in alphabetical order. Please, move accordingly.

Reply:
all_match should be the first item in alphabetical order, I double-checked the order in [presto doc](https://prestodb.io/docs/current/functions/array.html#:~:text=all_match(array(T)%2C-,function,-(T%2C%20boolean))).